keyprotect/sechandler/keyprotect.go

```diff
@@ -27,54 +26,34 @@ type keyprotectSecretKeyHandler struct {
 	kpClient *kp.Client
 }
 
-// keyprotectWrappedKey encapsulates the struct of the wrapped key as returned
-// by keyprotect. The example key looks like:
-// {
-//     "ciphertext": "eyJjBoZXJ.........0Z1YmYwIn0=",
-//     "keyVersion": {
-//         "id": "27b941a0-ab34-4b92-960d-30fa80f15bf0"
-//     }
-// }
-type keyprotectWrappedKey struct {
-	Ciphertext string `json:"ciphertext"`
-	KeyVersion struct {
-		Id string `json:"id"`
-	} `json:"keyVersion"`
-}
-
 // handleSecret unwraps the keys by calling the key protect unwrap service
 func (skh *keyprotectSecretKeyHandler) handleSecret(data map[string][]byte) (map[string][]byte, error) {
```

Comment:
seems like 
```suggestion
func (skh *keyprotectSecretKeyHandler) handleSecret(data map[string][]byte) ([]byte, error) {
```
would be better but understand that may not be compatible elsewhere, feel free to reject

Reply:
yea it is incompatible with other implementations since it is possible that a secret may return multiple keys. I will update the documentation of the implemented interface to make this clearer.

Reply:
fair